editor/frontend/src/EditTool.ts

```diff
@@ -44,8 +45,40 @@ export class EditTools {
     return IpcApp.callIpcChannel(editorChannel, "startCommand", commandId, iModelKey, ...args) as Promise<T>;
   }
 
-  public static async callCommand(methodName: string, ...args: any[]): Promise<any> {
-    return IpcApp.callIpcChannel(editorChannel, "callMethod", methodName, ...args);
+  public static async callCommand(
```

Comment:
not a fan of one argument per line.

Reply:
will undo formatting this.

Reply:
it can be helpful, for example if you include a comments about arguments. But, generally i think vertical screen space should be preserved where possible. My "old guy" programming philosophy...

Reply:
done